router/go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/tidwall/gjson v1.18.0
 	github.com/tidwall/sjson v1.2.5
 	github.com/twmb/franz-go v1.16.1
-	github.com/wundergraph/graphql-go-tools/v2 v2.0.0-rc.168
+	github.com/wundergraph/graphql-go-tools/v2 v2.0.0-rc.168.0.20250404210002-57dea1f1c1a8
```

Comment:
This doesn't look right

Reply:
It is right. We first need to release the engine. While this allows me to test it.